src/database/sql/sequelize/repositories/clinical/careplan/careplan.repo.ts

```diff
@@ -391,8 +406,7 @@ export class CareplanRepo implements ICareplanRepo {
 
             const foundResults = await CareplanActivity.findAndCountAll({
                 where : {
-                    Provider : "REAN",
-                    PlanName : "Maternity Careplan"
+                    Provider : "REAN"
```

Comment:
Why are you hard-coding Provider as REAN?

Reply:
Done